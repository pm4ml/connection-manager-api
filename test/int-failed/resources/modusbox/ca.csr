-----BEGIN CERTIFICATE REQUEST-----
MIIEmjCCAoICAQAwVTELMAkGA1UEBhMCVVMxCzAJBgNVBAgTAldBMRAwDgYDVQQH
EwdTZWF0dGxlMREwDwYDVQQKEwhNb2R1c2JveDEUMBIGA1UECxMLRW5naW5lZXJp
bmcwggIiMA0GCSqGSIb3DQEBAQUAA4ICDwAwggIKAoICAQDKYCMiV4lIvGbdClYG
bMXCXTdO+T4YCM/8kA/rIAYjnyhXbGrToFY1Hrj9Jz1myHxh946sVKDUNPf1VAlX
3G9cXIOSrVUYDnROYvF1DZRlAeOaiOn3D9PGzmGUeqTgg07vBICh7qjagvUsF2kP
IPYG0ZNy4MOnlSGjqHJdNDY3pd01BC1D8geJfEKTjAcqLbxY0gcF/bNLiP3sU0IJ
cJiAzYWtSK8snElFxHFSYy4lPZ4uEiWHol0BoGTUG7pPbT2ggqa8m3sjYCYBHi7M
vz74Fmhh+TZ1fXArK2Rfx1j27K8xm3kqEnsCI7TyzX0Ou7JwvsiqXj41sIWNgqF8
YKVV1ajPifJ8PHRfDECN4x7hecIohxLtnG2QWqAJb5DDFqWdbRyTx+QV1i7KMgeZ
OGophgDdLVxAiLbIlB8SGsVkHiBnrCIoxzAQUtV36d4QNKdEOFDn0tC4o4z+mzjl
K1oeBfZAg+bdC8ZmB985QNGfaDp/uog1Yp271v5HZ3M9sB/Aj+dD/700qRJMlzSX
ttNr63mwPrOWE1i2HYhFXerKIXnSf3Yy9bblY738MEsiRV3rNSF4xb0jcNWIwvIz
7yMzHB/vquzMfzwxsXkt1OgSuzeWj0UnydWtyi2N75O3Y2Gy9nJAKvyGzLPOla1K
t6UxWoMSbCUJNcLa7UAts52QUQIDAQABoAAwDQYJKoZIhvcNAQENBQADggIBAKhF
XIc9WZ5qPGn1VkpX+QIObPPZWAJycv6O/uzwqcZNHEw0ev0dbbRnu4Fxb1QuY+Er
cWqZAMX9x2odwttMW3mMAQA4mNzyejlj9DhkBhHna0tYz4gQR3An1GqjOFHSV7Ko
66u3dOFp4h9SIs/AuyiyOXmRlMGlbu44BdALBU4y2tnHaFPkXqNDUycdxFZ9772c
VRD33+XD3kzEmNsVy/Gl7juOPAZ7rPI7nM3o68Nl5xMlcHw9GB/+atzJkpsLSgmj
sw+G9ZfV9dEzKkpT1VxtwaVo1gXZsbxKdv4X0yicLEZ3g8mC5GXcy4U9PX2a4YeE
Yk+RP6aw/VzHUVFgH0mZb5cGze2mSkud6lNor28YmPnzScFs9BHB5eqhawkct/ld
/FcOXnhzkIpeBFFjxoCeQslcCyZVeyYer45L9vdL7VoRUAxDH6OB6XfDUcsVq5w3
SPc9ylZnFou12aJaz21twB22xtqQ4hMPyWp32P40Xsq+Mywoh4UxDpTw5nn3Mcht
/Csx/A+EEAbjWgypO98rSJnnuEC7Xs8CdIxJIcuvOZrNNctxrwyhyz4KtTi3VR1B
tZJmfFCWQlitriQtbG01wkVRdKfmRp+LTCEkrC90G2gtd5PwN3LcSIIQPvLH6iNt
aYm5ZDR98OcdqrmHVZ8ccpkdGUMIuqei90x1zTt6
-----END CERTIFICATE REQUEST-----
