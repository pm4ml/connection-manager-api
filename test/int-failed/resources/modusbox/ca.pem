-----BEGIN CERTIFICATE-----
MIIFejCCA2KgAwIBAgIUF2N8qOlDILCjrsb8WPJszgL6V9QwDQYJKoZIhvcNAQEN
BQAwVTELMAkGA1UEBhMCVVMxCzAJBgNVBAgTAldBMRAwDgYDVQQHEwdTZWF0dGxl
MREwDwYDVQQKEwhNb2R1c2JveDEUMBIGA1UECxMLRW5naW5lZXJpbmcwHhcNMTkw
NjA1MjM0MzAwWhcNMjQwNjAzMjM0MzAwWjBVMQswCQYDVQQGEwJVUzELMAkGA1UE
CBMCV0ExEDAOBgNVBAcTB1NlYXR0bGUxETAPBgNVBAoTCE1vZHVzYm94MRQwEgYD
VQQLEwtFbmdpbmVlcmluZzCCAiIwDQYJKoZIhvcNAQEBBQADggIPADCCAgoCggIB
AMpgIyJXiUi8Zt0KVgZsxcJdN075PhgIz/yQD+sgBiOfKFdsatOgVjUeuP0nPWbI
fGH3jqxUoNQ09/VUCVfcb1xcg5KtVRgOdE5i8XUNlGUB45qI6fcP08bOYZR6pOCD
Tu8EgKHuqNqC9SwXaQ8g9gbRk3Lgw6eVIaOocl00Njel3TUELUPyB4l8QpOMByot
vFjSBwX9s0uI/exTQglwmIDNha1IryycSUXEcVJjLiU9ni4SJYeiXQGgZNQbuk9t
PaCCprybeyNgJgEeLsy/PvgWaGH5NnV9cCsrZF/HWPbsrzGbeSoSewIjtPLNfQ67
snC+yKpePjWwhY2CoXxgpVXVqM+J8nw8dF8MQI3jHuF5wiiHEu2cbZBaoAlvkMMW
pZ1tHJPH5BXWLsoyB5k4aimGAN0tXECItsiUHxIaxWQeIGesIijHMBBS1Xfp3hA0
p0Q4UOfS0LijjP6bOOUrWh4F9kCD5t0LxmYH3zlA0Z9oOn+6iDVinbvW/kdncz2w
H8CP50P/vTSpEkyXNJe202vrebA+s5YTWLYdiEVd6sohedJ/djL1tuVjvfwwSyJF
Xes1IXjFvSNw1YjC8jPvIzMcH++q7Mx/PDGxeS3U6BK7N5aPRSfJ1a3KLY3vk7dj
YbL2ckAq/IbMs86VrUq3pTFagxJsJQk1wtrtQC2znZBRAgMBAAGjQjBAMA4GA1Ud
DwEB/wQEAwIBBjAPBgNVHRMBAf8EBTADAQH/MB0GA1UdDgQWBBQJP5EtJFhtyMYk
yJ+NSnYb3qsl+DANBgkqhkiG9w0BAQ0FAAOCAgEAeAEX0MRuGCrPgvSBChx9yD/g
Uzlwf8nYi+aQqloMWcpskwEwV/KsVSsfG7xVID9+TBaundP0/QEpFlB0wRdpItEV
lWrDM8MDPzgMSB4oByRDyN3WKJIa/OS7L9FvTQsJcD0fgwv9hQIV138l0qKvwGDh
X8iLvxr5UeDUiF0S/KhYx09bwcZqij4XKHTOQ4mcRMDwFPbym+rQcEgGW86dSGl1
tP18w5fArzBHUtBuIMp6o7l6c/sMLASvrewnIVxeD1MjklGP6G9i1s8lvNhB327k
sfZ9nB42kEuq75siKLGZjBNqxuIeJnHUvNVZHtjvXuhOQJ9RWfncqtbvw1IahNtU
/mxA86J7iXQpweZcQBoh0Ua5fWIsbHxQnc97PS7+DoRNYrSKt+einAngHczB4lcp
Z5AuApBoOxTy6yHmI7eIEfqjiBRrnXgr8CgBjaaqDIeNHzJOxckrZXTWJbbpmvBo
D5k5j0ZiBpwoQeyNcxmpFo3JGqyi/D51ckum8z75IE3kOWXqki4Q81qsnzig5G3Y
pDV4fP1hC1bgqDD7aGmPMRCxt+gqaqB3Wbgm/lDugu/oWi6qpeptMwls5RK6u2vo
UfvHXEZSuM1l8DL1veKuyhSz27GJ8oPajI4yl12lS3JNJoCQvX16u2g2xsITg3y7
3jXPJm4yx3GNh0DZcN4=
-----END CERTIFICATE-----
