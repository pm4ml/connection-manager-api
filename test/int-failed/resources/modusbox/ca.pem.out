Certificate:
    Data:
        Version: 3 (0x2)
        Serial Number:
            17:63:7c:a8:e9:43:20:b0:a3:ae:c6:fc:58:f2:6c:ce:02:fa:57:d4
    Signature Algorithm: sha512WithRSAEncryption
        Issuer: C=US, ST=WA, L=Seattle, O=Modusbox, OU=Engineering
        Validity
            Not Before: Jun  5 23:43:00 2019 GMT
            Not After : Jun  3 23:43:00 2024 GMT
        Subject: C=US, ST=WA, L=Seattle, O=Modusbox, OU=Engineering
        Subject Public Key Info:
            Public Key Algorithm: rsaEncryption
                Public-Key: (4096 bit)
                Modulus:
                    00:ca:60:23:22:57:89:48:bc:66:dd:0a:56:06:6c:
                    c5:c2:5d:37:4e:f9:3e:18:08:cf:fc:90:0f:eb:20:
                    06:23:9f:28:57:6c:6a:d3:a0:56:35:1e:b8:fd:27:
                    3d:66:c8:7c:61:f7:8e:ac:54:a0:d4:34:f7:f5:54:
                    09:57:dc:6f:5c:5c:83:92:ad:55:18:0e:74:4e:62:
                    f1:75:0d:94:65:01:e3:9a:88:e9:f7:0f:d3:c6:ce:
                    61:94:7a:a4:e0:83:4e:ef:04:80:a1:ee:a8:da:82:
                    f5:2c:17:69:0f:20:f6:06:d1:93:72:e0:c3:a7:95:
                    21:a3:a8:72:5d:34:36:37:a5:dd:35:04:2d:43:f2:
                    07:89:7c:42:93:8c:07:2a:2d:bc:58:d2:07:05:fd:
                    b3:4b:88:fd:ec:53:42:09:70:98:80:cd:85:ad:48:
                    af:2c:9c:49:45:c4:71:52:63:2e:25:3d:9e:2e:12:
                    25:87:a2:5d:01:a0:64:d4:1b:ba:4f:6d:3d:a0:82:
                    a6:bc:9b:7b:23:60:26:01:1e:2e:cc:bf:3e:f8:16:
                    68:61:f9:36:75:7d:70:2b:2b:64:5f:c7:58:f6:ec:
                    af:31:9b:79:2a:12:7b:02:23:b4:f2:cd:7d:0e:bb:
                    b2:70:be:c8:aa:5e:3e:35:b0:85:8d:82:a1:7c:60:
                    a5:55:d5:a8:cf:89:f2:7c:3c:74:5f:0c:40:8d:e3:
                    1e:e1:79:c2:28:87:12:ed:9c:6d:90:5a:a0:09:6f:
                    90:c3:16:a5:9d:6d:1c:93:c7:e4:15:d6:2e:ca:32:
                    07:99:38:6a:29:86:00:dd:2d:5c:40:88:b6:c8:94:
                    1f:12:1a:c5:64:1e:20:67:ac:22:28:c7:30:10:52:
                    d5:77:e9:de:10:34:a7:44:38:50:e7:d2:d0:b8:a3:
                    8c:fe:9b:38:e5:2b:5a:1e:05:f6:40:83:e6:dd:0b:
                    c6:66:07:df:39:40:d1:9f:68:3a:7f:ba:88:35:62:
                    9d:bb:d6:fe:47:67:73:3d:b0:1f:c0:8f:e7:43:ff:
                    bd:34:a9:12:4c:97:34:97:b6:d3:6b:eb:79:b0:3e:
                    b3:96:13:58:b6:1d:88:45:5d:ea:ca:21:79:d2:7f:
                    76:32:f5:b6:e5:63:bd:fc:30:4b:22:45:5d:eb:35:
                    21:78:c5:bd:23:70:d5:88:c2:f2:33:ef:23:33:1c:
                    1f:ef:aa:ec:cc:7f:3c:31:b1:79:2d:d4:e8:12:bb:
                    37:96:8f:45:27:c9:d5:ad:ca:2d:8d:ef:93:b7:63:
                    61:b2:f6:72:40:2a:fc:86:cc:b3:ce:95:ad:4a:b7:
                    a5:31:5a:83:12:6c:25:09:35:c2:da:ed:40:2d:b3:
                    9d:90:51
                Exponent: 65537 (0x10001)
        X509v3 extensions:
            X509v3 Key Usage: critical
                Certificate Sign, CRL Sign
            X509v3 Basic Constraints: critical
                CA:TRUE
            X509v3 Subject Key Identifier: 
                09:3F:91:2D:24:58:6D:C8:C6:24:C8:9F:8D:4A:76:1B:DE:AB:25:F8
    Signature Algorithm: sha512WithRSAEncryption
         78:01:17:d0:c4:6e:18:2a:cf:82:f4:81:0a:1c:7d:c8:3f:e0:
         53:39:70:7f:c9:d8:8b:e6:90:aa:5a:0c:59:ca:6c:93:01:30:
         57:f2:ac:55:2b:1f:1b:bc:55:20:3f:7e:4c:16:ae:9d:d3:f4:
         fd:01:29:16:50:74:c1:17:69:22:d1:15:95:6a:c3:33:c3:03:
         3f:38:0c:48:1e:28:07:24:43:c8:dd:d6:28:92:1a:fc:e4:bb:
         2f:d1:6f:4d:0b:09:70:3d:1f:83:0b:fd:85:02:15:d7:7f:25:
         d2:a2:af:c0:60:e1:5f:c8:8b:bf:1a:f9:51:e0:d4:88:5d:12:
         fc:a8:58:c7:4f:5b:c1:c6:6a:8a:3e:17:28:74:ce:43:89:9c:
         44:c0:f0:14:f6:f2:9b:ea:d0:70:48:06:5b:ce:9d:48:69:75:
         b4:fd:7c:c3:97:c0:af:30:47:52:d0:6e:20:ca:7a:a3:b9:7a:
         73:fb:0c:2c:04:af:ad:ec:27:21:5c:5e:0f:53:23:92:51:8f:
         e8:6f:62:d6:cf:25:bc:d8:41:df:6e:e4:b1:f6:7d:9c:1e:36:
         90:4b:aa:ef:9b:22:28:b1:99:8c:13:6a:c6:e2:1e:26:71:d4:
         bc:d5:59:1e:d8:ef:5e:e8:4e:40:9f:51:59:f9:dc:aa:d6:ef:
         c3:52:1a:84:db:54:fe:6c:40:f3:a2:7b:89:74:29:c1:e6:5c:
         40:1a:21:d1:46:b9:7d:62:2c:6c:7c:50:9d:cf:7b:3d:2e:fe:
         0e:84:4d:62:b4:8a:b7:e7:a2:9c:09:e0:1d:cc:c1:e2:57:29:
         67:90:2e:02:90:68:3b:14:f2:eb:21:e6:23:b7:88:11:fa:a3:
         88:14:6b:9d:78:2b:f0:28:01:8d:a6:aa:0c:87:8d:1f:32:4e:
         c5:c9:2b:65:74:d6:25:b6:e9:9a:f0:68:0f:99:39:8f:46:62:
         06:9c:28:41:ec:8d:73:19:a9:16:8d:c9:1a:ac:a2:fc:3e:75:
         72:4b:a6:f3:3e:f9:20:4d:e4:39:65:ea:92:2e:10:f3:5a:ac:
         9f:38:a0:e4:6d:d8:a4:35:78:7c:fd:61:0b:56:e0:a8:30:fb:
         68:69:8f:31:10:b1:b7:e8:2a:6a:a0:77:59:b8:26:fe:50:ee:
         82:ef:e8:5a:2e:aa:a5:ea:6d:33:09:6c:e5:12:ba:bb:6b:e8:
         51:fb:c7:5c:46:52:b8:cd:65:f0:32:f5:bd:e2:ae:ca:14:b3:
         db:b1:89:f2:83:da:8c:8e:32:97:5d:a5:4b:72:4d:26:80:90:
         bd:7d:7a:bb:68:36:c6:c2:13:83:7c:bb:de:35:cf:26:6e:32:
         c7:71:8d:87:40:d9:70:de
